pkg/controller/jaeger/account.go

```diff
@@ -22,21 +23,30 @@ func (r *ReconcileJaeger) applyAccounts(jaeger v1.Jaeger, desired []corev1.Servi
 
 	inv := inventory.ForAccounts(list.Items, desired)
 	for _, d := range inv.Create {
-		jaeger.Logger().WithField("account", d.Name).Debug("creating service account")
+		jaeger.Logger().WithFields(log.Fields{
+			"account":   d.Name,
+			"namespace": d.Namespace,
+		}).Debug("creating service account")
 		if err := r.client.Create(context.Background(), &d); err != nil {
 			return err
 		}
 	}
 
 	for _, d := range inv.Update {
-		jaeger.Logger().WithField("account", d.Name).Debug("updating service account")
+		jaeger.Logger().WithFields(log.Fields{
+			"account":   d.Name,
+			"namespace": d.Namespace,
+		}).Debug("updating service account")
 		if err := r.client.Update(context.Background(), &d); err != nil {
 			return err
 		}
 	}
 
 	for _, d := range inv.Delete {
-		jaeger.Logger().WithField("account", d.Name).Debug("deleting service account")
+		jaeger.Logger().WithFields(log.Fields{
```

Comment:
This overrides the namespace from the `jaeger.Logger()`. Before this change, the log would say that it "deleting service account" in namespace `tenant1` when in fact the SA was in the namespace `tenant2`.